kedro-airflow/kedro_airflow/plugin.py

```diff
@@ -28,26 +28,43 @@ def airflow_commands():
 @click.option("-e", "--env", default="local")
 @click.option(
     "-t",
-    "--target-dir",
+    "--target-path",
     "target_path",
-    type=click.Path(writable=True, resolve_path=True, file_okay=False),
-    default="./airflow_dags/",
+    default="./airflow_dags/{package_name}_dag.py",
+)
+@click.option(
+    "-j",
+    "--jinja-file",
+    "jinja_template",
```

Comment:
```suggestion
```
Small simplification: you don't need to name this variable explicitly if you just change to using `jinja_file` in the function.